_tests/browser_tests/README.md

```diff
@@ -0,0 +1,92 @@
+## Quick start:
+
+_Note: If you haven't, make sure to run `./setup.sh from the project root` to install Protractor globally._
+
+From the project root:
+
+```sh
+# Update Selenium Server binaries:
+webdriver-manager update
+```
+
+From this directory:
+
+```sh
+# Start Selenium Server:
+webdriver-manager start
+
+# Open a new tab and tell protractor to start testing:
+protractor conf.js
+```
+
+If you want to test a server other than your local instance (assuming port 7000 is your local CFGov server) use the `--baseUrl` flag. Ex:
+
+```sh
+protractor conf.js --baseUrl=http://beta.consumerfinance.gov
+```
+
+## Pages
+
+Pages are organized in `page_objects`. To add new tests start with adding a new page file and include the elements you'll want to query the page for. For example, if you wanted to test the page title and all the `h2` elements on the about page you'd write:
+
+```js
+var TheBureauPage = function() {
```

Comment:
Presently in the project JS we're favoring function declarations over expressions (`function TheBureauPage()` vs `var TheBureauPage = function()`). Declarations get hoisted to the top of the execution context—some say that makes the code misleading. Expressions create a variable that references an anonymous function—some say that makes it harder to debug. At any rate, making the style consistent between project and test code may be preferable.